### ISSUE TITLE
Update repo?

### ISSUE THREAD

**Issue by dj2** [PDFium/PDFium#2]
The PDFium repo is quite a ways out of date. Would it be possible to either get the repo updated, or get access so I can update the repo myself?

**Comment by dj2** [comment]
Ping.

**Comment by dj2** [comment]
ping.

**Comment by dj2** [comment]
Ping, any chance to get this updated, or control passed off so it can be updated?

**Comment by dj2** [comment]
ping.

**Comment by dj2** [comment]
Ping. Any chance we can update this?

**Comment by koppor** [comment]
Currently, the official repository seems to be https://pdfium.googlesource.com/pdfium/.

A maintained project building on PDFium is https://github.com/pvginkel/PdfiumViewer.

**Comment by dj2** [comment]
Yes, but the github mirror showed high in search ranking and it's 3 years
out of date.  I'd like to sync it to tip of tree but there is no response
from the creator.

Dan



On Aug 24, 2017 01:57, "Oliver Kopp" <notifications@github.com> wrote:

Currently, the official repository seems to be https://pdfium.googlesource.
com/pdfium/.

A maintained project building on PDFium is https://github.com/pvginkel/
PdfiumViewer.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/PDFium/PDFium/issues/2#issuecomment-324540896>, or mute
the thread

**Comment by nicolas-raoul** [comment]
The owner of this repository should either:
- Set up mirroring from the official repo,
- Mention prominently in the README how it is different from the official repo and why (for instance they stopped at commit X because of reason Y),
- or delete the repo.

**Comment by dj2** [comment]
I've been keeping a mirror of the official repo at https://github.com/dj2/pdfium. It's usually no more then a week behind.

**Comment by dj2** [comment]
ping?
